marketplace/deployer_util/config_helper_test.py

```diff
@@ -1038,15 +1038,26 @@ def test_resource_constraints(self):
                 simpleNodeAffinity:
                   type: REQUIRE_ONE_NODE_PER_REPLICA
             - replicas: 5
+              requests:
+                cpu: 50m
               affinity:
                 simpleNodeAffinity:
                   type: REQUIRE_MINIMUM_NODE_COUNT
                   minimumNodeCount: 4
+            - requests:
+                cpu: 200m
```

Comment:
rm?

Reply:
Remove `cpu` spec--we're don't have the ability to fit nodes that request both cpu and gpu.

Reply:
Done, as a part of disallowing cpu/memory constraints if GPU requests are specified.